lib/ansible/modules/cloud/amazon/_ec2_vpc.py

```diff
@@ -74,7 +74,7 @@
       - Tags in this list are used in conjunction with CIDR block to uniquely identify a VPC in lieu of vpc_id. Therefore,
         if CIDR/Tag combination does not exist, a new VPC will be created.  VPC tags not on this list will be ignored. Prior to 1.7,
         specifying a resource tag was optional.'
```

Comment:
Probably should add something to the docs. `Prior to 1.7 and after ...`? Will this be backported?